app/stacks/global/front-door/waf-policy.tf

```diff
@@ -169,3 +169,188 @@ resource "azurerm_frontdoor_firewall_policy" "default" {
+          selector       = "cookie_policy"
+        }
+      }
+    }
+
+    override {
+      rule_group_name = "SQLI"
+
+      rule {
+        # Common SQL injection testing
+        rule_id = "942110"
+        action  = "Redirect"
+      }
+
+      rule {
+        # Detects MySQL comment-/space-obfuscated injections and backtick termination
+        rule_id = "942200"
+        action  = "Block"
+
+        exclusion {
+          # Exclusion to allow acceptance of cookies
+          match_variable = "RequestCookieNames" # "CookieValue:cookie_policy"
+          operator       = "Equals"
+          selector       = "cookie_policy"
+        }
+      }
+
+      rule {
+        # Detects basic SQL authentication bypass attempts 2/3
+        rule_id = "942260"
+        action  = "Block"
+
+        exclusion {
+          # Exclusion to allow acceptance of cookies
+          match_variable = "RequestCookieNames" # "CookieValue:cookie_policy"
+          operator       = "Equals"
+          selector       = "cookie_policy"
+        }
+      }
+
+      rule {
+        # Suspisious use of SQL keywords
+        rule_id = "942400"
+        action  = "Redirect"
+      }
+
+      rule {
+        # Restricted SQL Character Anomaly Detection (args): # of special characters exceeded (12)
+        rule_id = "942430"
+        action  = "Block"
+
+        exclusion {
+          # Exclusion to allow Azure AD authentication redirection
+          match_variable = "QueryStringArgNames" # "QueryParamValue:code"
+          operator       = "Equals"
+          selector       = "code"
+        }
+      }
+
+      rule {
+        # SQL Comment Sequence Detected
+        rule_id = "942440"
+        action  = "Block"
+
+        exclusion {
+          # Exclusion to allow Azure AD authentication redirection
+          match_variable = "QueryStringArgNames" # "QueryParamValue:code"
+          operator       = "Equals"
+          selector       = "code"
+        }
+      }
+
+      rule {
+        # SQL Hex Encoding Identified
+        rule_id = "942450"
+        action  = "Block"
+
+        exclusion {
+          # Exclusion to allow cookie connect.sid
+          match_variable = "RequestCookieNames" # "CookieValue:connect.sid"
+          operator       = "Equals"
+          selector       = "connect.sid"
+        }
+      }
+    }
+
+    # Exception for ASB-2059 - Exclude all rules for this selector.
+    exclusion {
+      match_variable = "RequestBodyPostArgNames"
+      operator       = "Equals"
+      selector       = "comment"
+    }
+
+    # Exception for ASB-1692 merged with ASB-1928 - Exclude all rules for this selector.
+    # POST project update content, which is a strict subset of HTML
+    # only applies Back Office, so should be removed from others with new Front Door
+    exclusion {
+      match_variable = "RequestBodyPostArgNames"
+      operator       = "Equals"
+      selector       = "backOfficeProjectUpdateContent"
+    }

```

Comment:
We can remove this exclusions, since they are for applications + back office respectively.